google/cloud/security/inventory/pipeline_builder.py

```diff
@@ -90,8 +90,8 @@ def _get_api(self, api_name):
                 else:
                     api = api_class(self.global_configs,
                                     version=api_version)
-            except api_errors.ApiExecutionError as e:
-                LOGGER.error('Failed to execute API %s, v=%s\n%s',
+            except api_errors.ApiInitializationError as e:
```

Comment:
I think you should have an except for both ApiExecutionError and ApiInitializationError. That way you can recover or propagate either type of error.

Reply:
I would do it, but at this point, we really are just initializing the api clients and not executing them.  Can you please confirm if this is still what would be nice to do?

Changed the raise statement to just a single ```raise```.

Reply:
So long as that's the only exception that we care about at this stage and we know we've consistently made sure that's the exception that is raised, then that makes sense. I think longer term there is a lot more work to do around exception handling and ensuring all APIs are consistent. I tried to make things more consistent in my mixins PR, but I know more work needs to be done.